lib/kafo/data_type_parser.rb

```diff
@@ -15,7 +15,7 @@ def initialize(manifest)
         line = line.force_encoding("UTF-8").strip
         next if line.start_with?('#')
 
-        line = line.split(' #').first.strip
+        line = line.split(' #').first.strip unless line.empty?
```

Comment:
I had to check this, but what ends up happening is `''.split(' #')` returns `[]` so `[].first` is `nil`.